source/MaterialXTest/RenderGLSL.cpp

```diff
@@ -88,20 +88,10 @@ void GlslShaderRenderTester::loadLibraries(mx::DocumentPtr document,
                                            RenderUtil::RenderTestOptions& options)
 {
     mx::FilePath lightDir = mx::FilePath::getCurrentPath() / mx::FilePath("resources/Materials/TestSuite/Utilities/Lights");
-    if (options.lightFiles.size() == 0)
```

Comment:
Only load lights if set by the options. Otherwise we can't disable direct lights.